docs/coding-guidelines/css.md

```diff
@@ -146,14 +146,15 @@ Under the hood, we are using webpack and its `sass-loader`, for compiling the st
 To avoid code bloat and have a more consistent experience we use the same breakpoints in all SCSS files whenever possible. DO NOT define your own media queries. We should use the `break-*` mixins from [Gutenberg](https://github.com/WordPress/gutenberg/blob/0f1f5e75408705f0ec014f5d2ea3d9fcc8a97817/packages/base-styles/_mixins.scss). For example:
 
 ```scss
-@import "@wordpress/base-styles/breakpoints";
-@import "@wordpress/base-styles/mixins";
+@use "@wordpress/base-styles/colors";
+@use "@wordpress/base-styles/mixins";
```

Comment:
I'm curious how we'd feel about using `as *` to avoid the namespacing prefixing changes you added below, i.e. continue to reference variables as `$gray-900` or `break-mobile`.

Personally, I think namespaces could be helpful and encourage us to be more explicit, clearer in where variables come from, and in what we're using and limit how much we pollute into the file-specific global namespace. But `as *` is less verbose, and may be easier to achieve in a bulk transition from legacy and modern syntax.

```suggestion
@use "@wordpress/base-styles/colors" as *;
@use "@wordpress/base-styles/mixins" as *;
```